src/views/Start/Start.js

```diff
@@ -42,13 +47,6 @@ function Start() {
       : 'Listen & Speak'
   );
 
-  const [sel_lang_text, set_sel_lang_text] = useState(
```

Comment:
this is not required?

Reply:
We have same state like this. this is not using anywhere So removed it. Don't worry Code is not impacted.